features/karaf/esh-core/src/main/history/dependencies.xml

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Please change `features/karaf*/src/main/history` to `features/karaf/*/src/main/history` in `.gitignore` and don't add files that are placed in that directory.